assignment.js

```diff
@@ -112,7 +119,24 @@ Use the following test cases to confirm your program meets the success criteria
 console.log('Problem 3:')
 
 // Add your code below this line
-
+let playerAChoice = "rock"
+let playerBChoice = "rock"
+
+if (playerAChoice === playerBChoice) {
+  console.log("tie")
+} else if (playerAChoice === "rock" && playerBChoice === "scissors")  {
+  console.log("playerA wins")
+} else if (playerAChoice === "rock" && playerBChoice === "paper") {
+  console.log("playerB wins")
+} else if (playerAChoice === "scissors" && playerBChoice === "rock")  {
+  console.log("playerB wins")
+} else if (playerAChoice === "scissors" && playerBChoice === "paper") {
+  console.log("playerA wins")
+} else if (playerAChoice === "paper" && playerBChoice === "rock") {
+  console.log("playerA wins")
+} else if(playerAChoice === "paper" && playerBChoice === "scissors")  {
+  console.log("playerB wins")
+}
 // Add your code above this line
 
```

Comment:
@laraann88 Good Job utilizing the `if` `else` `else if`. Your code checks for the right outcomes and it evaluates the right outcome based on input. I would suggest adding one more else statement that would `console.log` an error message if `playerAChoice` and `playerBChoice` choices aren't correct it would console.log an error. Take a look at this approach [here](https://github.com/jsr20190826/hw-01-js-basics-control-flow-loops-solution/blob/a810fe3f581584e5015519a3508b078d38c4c8c9/assignment.js#L150)